app/src/main/java/org/jellyfin/androidtv/util/BackgroundManagerUtils.kt

```diff
@@ -0,0 +1,36 @@
+package org.jellyfin.androidtv.util
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.leanback.app.BackgroundManager
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+import timber.log.Timber
+
+fun BackgroundManager.drawable(
+	context: Context,
+	url: String,
+	widthPixels: Int,
+	heightPixels: Int
+) {
+
+	Glide.with(context)
+		.load(url)
+		.override(widthPixels, heightPixels)
```

Comment:
```suggestion
		.override(widthPixels, heightPixels)
		.centerCrop()
```

When testing I noticed some backdrops getting stretched which did not happen before this PR. Adding the centerCrop function here fixed it.

Reply:
And it makes sense because I eliminated it without realizing